### ISSUE TITLE
Failing to create service in heroku-type deployment

### ISSUE THREAD

**Issue by blysik** [ustream/openduty#42]
I have it deployed in a Heroku-like system. (Uses Heroku buildpacks, generally behaves like Heroku.) Everything looks good.  

However, when I try to save a new Service, I get this error:
```
DataError at /services/save
invalid input syntax for uuid: "0"
LINE 1: ... FROM "openduty_service" WHERE "openduty_service"."id" = '0'
```

In the Postgres log:
```
ERROR:  invalid input syntax for uuid: "0" at character 257
STATEMENT:  SELECT "openduty_service"."name", "openduty_service"."id", "openduty_service"."retry", "openduty_service"."policy_id", "openduty_service"."escalate_after", "openduty_service"."notifications_disabled" FROM "openduty_service" WHERE "openduty_service"."id" = '0'
```

Any ideas?

**Comment by blysik** [comment]
Perhaps related:
https://github.com/dcramer/django-uuidfield/issues/46

**Comment by blysik** [comment]
It feels like openduty is trying to save the Service with an id of '0', but since postgres doesn't think '0' is a proper UUID, it fails?

**Comment by blysik** [comment]
And related: http://stackoverflow.com/questions/20928439/getting-invalid-input-syntax-for-uuid-when-using-django-uuidfield-as-primar

**Comment by deathowl** [comment]
Wow, amazing bugreport, looking right into the issue.

**Comment by deathowl** [comment]
The commit above should fix your issue, could you please confirm, that it did? If so, please close this issue @blysik

**Comment by deathowl** [comment]
Also, if you use a heroku-like env to run openduty, don't forget to launch your workers also. You could add the required commands to launch celeryd to the procfile

**Comment by blysik** [comment]
Deploying the fix now.  Here's what I've added to my Procfile for the worker stuff:
```
celery: celery worker -A openduty --loglevel=INFO -Ofair
scheduler: celery worker -B -A openduty --loglevel=INFO -Ofair
```
That look fine?

**Comment by deathowl** [comment]
yeah, totally

**Comment by blysik** [comment]
Looks like a different error now.  Here's the error from the postgres log side:
```
ERROR:  invalid input syntax for uuid: "" at character 175
STATEMENT:  UPDATE "openduty_service" SET "name" = 'SEO Standard', "retry" = 10, "policy_id" = 1, "escalate_after" = 30, "notifications_disabled" = false WHERE "openduty_service"."id" = ''
```

**Comment by blysik** [comment]
The error page is showing it to be line 90, at services.py.  That service.save()

**Comment by deathowl** [comment]
Thanks, will check that overnight.

**Comment by blysik** [comment]
Any other data I can provide to help troubleshoot this?

**Comment by deathowl** [comment]
Thanks, data provided is enough, i think i'll find the time tomorrow to look into it, sorry for the delay of things, I'm pretty busy at the moment.

**Comment by deathowl** [comment]
BTW i tested it on Heroku, and for me my patch DID work with postgres without any errors.

**Comment by blysik** [comment]
No problem at all!  I appreciate you looking at it.  

I wonder if it's a versioning issue of my postgres.

**Comment by deathowl** [comment]
HI! I just got back to this, Is it still actual @blysik ? If yes dont hesitate to ask around

**Comment by blysik** [comment]
Yes, I just checked, and that error still happens when I try to save a service.

**Comment by carxwol** [comment]
It's not happening anymore without django-uuidfield requirement.
UUIDField is integrated into Django 1.8, so i just removed it :
https://docs.djangoproject.com/en/1.8/ref/models/fields/#uuidfield
